src/chigame/api/tests.py

```diff
@@ -0,0 +1,136 @@
+# from django.test import TestCase
+
+# Create your tests here.
+
+# Compare this snippet from src/chigame/api/tests.py:
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from chigame.games.models import Game
+
+
+class GameTests(APITestCase):
+    def check_equal(self, object, expected: dict):
+        """
+        Helper function to check that the object data matches the expected data.
+        """
+
+        for key in expected:
+            self.assertEqual(getattr(object, key), expected[key])
+
+    def test_create_game(self):
+        """
+        Ensure we can create a new game object.
+        """
+
+        # Create a game object
+        url = reverse("api-game-list")
+        data = {"name": "The Witcher 3", "description": "Geralt of Rivia", "min_players": 1, "max_players": 4}
```

Comment:
Small note, for the sake of readability you might want to put each KV pair on a new line:

In general this is more readable and easily extensible since you can just add a new line, instead of having an ever expanding single line. 

This will matter much more as the `Game` model gets updated with new attributes being added

```
data = {
  "name": "The Witcher 3", 
  "description": "Geralt of Rivia", 
  "min_players": 1, 
  "max_players": 4
  }
```